### PR TITLE
Create pull request for Sprint48/trandb/feature/create-board-ui branch

### PR DESCRIPTION
**Pull request by Btrnn**
@irisnikita Dear Sir, please check my pull request. Thanks.

### CODE REVIEW

File: frontend/src/components/common/AccessDropdown/index.tsx [hunk 1]
```diff
@@ -0,0 +1,174 @@
+// Libraries
+import React, { useEffect, useMemo, useState } from 'react';
+
+// Components
+import { Modal, Input, List, Select, message, Tag, AutoComplete, Typography } from '../../ui';
+import { AddIcon, CloseIcon, DeleteIcon, SwitchUserIcon } from '../../icons';
+
+// Constants
+import { PERMISSION, ROLE_KEY, ROLE_OPTIONS } from 'constants/role';
+import {
```
Comment (irisnikita) [inline frontend/src/components/common/AccessDropdown/index.tsx:12]: @Btrnn Wrong position, should put in // Services group

File: frontend/src/components/common/shareAccessModal/index.tsx [hunk 2]
```diff
@@ -519,40 +503,14 @@ export const ShareAccessModal: React.FC<ShareAccessProp> = props => {
...
-
-                        return (
-                          <Text ellipsis={{ tooltip: true }} className={`${className} w-full`}>
-                            {Icon ? <Icon className="mr-2 my-2 shrink-0" /> : null}
-                            {label}
-                          </Text>
-                        );
+                    <AccessDropDown
+                      onSelect={role => {
+                        console.log(role);
```
Comment (irisnikita) [inline frontend/src/components/common/shareAccessModal/index.tsx]: @Btrnn Remove console.log please !!!!

File: frontend/src/modules/Dashboard/index.tsx [hunk 3]
```diff
@@ -679,6 +687,30 @@ export const Dashboard: React.FC = () => {
 
   // Utils
   const levelKeys = getDashBoardLevelKeys(dashBoardItems as LevelKeysProps[]);
+  const pathList = getParentKeys(dashBoardItems as LevelKeysProps[]);
+
+  const onOpenChange: MenuProps["onOpenChange"] = (openKeys) => {
+    const currentOpenKey = openKeys.find((key) => openList.indexOf(key) === -1);
+    console.log("🚀 ~ currentOpenKey:", currentOpenKey)
```
Comment (irisnikita) [inline frontend/src/modules/Dashboard/index.tsx]: @Btrnn Remove console.log

**Review by irisnikita — COMMENTED** [review]
@Btrnn Hi Tran, please check my comments

File: frontend/src/components/common/AccessDropdown/index.tsx (hunk 1 above)
Comment (Btrnn) [reply frontend/src/components/common/AccessDropdown/index.tsx:12]: @irisnikita i fixed, check again.

Comment (irisnikita) [reply frontend/src/components/common/AccessDropdown/index.tsx:12]: @Btrnn Oke

File: frontend/src/components/common/shareAccessModal/index.tsx (hunk 2 above)
Comment (Btrnn) [reply frontend/src/components/common/shareAccessModal/index.tsx]: @irisnikita I fixed.

Comment (irisnikita) [reply frontend/src/components/common/shareAccessModal/index.tsx]: @Btrnn oke

**Pull request merged**